core/arch/arm/mm/mobj.c

```diff
@@ -181,15 +181,8 @@ static void *mobj_virt_get_va(struct mobj *mobj, size_t offset)
 	return (void *)(vaddr_t)offset;
 }
 
-static void mobj_virt_free(struct mobj *mobj)
-{
-	mobj_virt_assert_type(mobj);
-	free(mobj);
-}
-
```

Comment:
minor: `mobj_virt_ops` and `mobj_virt_get_va` are not used. `mobj_virt` is based on `mobj_phys_ops`.